client/src/components/EventCard.js

```diff
@@ -14,11 +14,28 @@ const Card = styled.div`
   height: 93px;
   border-radius: 12px;
```

Comment:
Also this is not part of your change, you should keep in mind to not work with fixed PX's

Reply:
dito